tools/topology/topology2/include/common/tokens.conf

```diff
@@ -120,9 +120,9 @@ Object.Base.VendorToken {
 
 	 "12" {
 		name "sof_tkn_process"
-		# Token payload_with_output_fmt specifies whether there is
-		# output audio format in the init instance ipc4 message payload.
-		payload_with_output_fmt         901
+		# The token init_payload_format is used to describe the
+		# module instance initialization payload format
+		init_payload_format         901
```

Comment:
Allowing for multiple types of init payload is something we seem to need, so this seems more practical approach. It would be nice to have a pointer where the enumerated options are documented. This PR now defines "1".

Reply:
Yes, fixed now.